src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,23 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
```

Comment:
Destructuring improves the code. Thanks to the line:
`const { title, body, user, comments } = post;
`the code becomes more readable and easier to understand, while also avoiding duplication.